README.md

```diff
@@ -86,37 +86,41 @@ tests run on nodejs and you'll need nodejs>=v0.12 and npm>=v2.0 installed.
 Also for the time being, shared code for tests lives in plugins at
 `tests/jsmarionette/plugins` or in helpers at `shared/test/integration`.
 
+For more details on writing integration tests, see:
+https://developer.mozilla.org/en-US/docs/Mozilla/Firefox_OS/Automated_testing/Gaia_integration_tests
+
 #### Running integration tests
 
 ```sh
-npm run marionette
+make test-integration
 ```
 
-#### Invoking specific test files
+#### Invoking tests for a specific app
 
 ```sh
-TEST_FILES="/abs/path/to/some_test.js /abs/path/to/other_test.js" npm run marionette
+APP=<APP> make test-integration
 ```
 
-#### Invoking tests for a specific app
+#### Invoking specific test files
 
 ```sh
-APP=<APP> npm run marionette
+TEST_FILES="/abs/path/to/some_test.js /abs/path/to/other_test.js" make test-integration
 ```
 
-#### Skipping a test file
+##### Running tests on device
 
+You can run tests on device by plugging in your phone and adding the BUILDAPP=device to the make command:
 ```sh
-SKIP_TEST_FILES="/abs/path/to/skipped_test.js /abs/path/to/other/skipped_test.js" npm run marionette
+BUILDAPP=device make test-integration
 ```
 
 #### Moar things
 
 + `VERBOSE=1` pipes gecko logs to your command line process for debugging.
 
-+ If you don't have a b2g/ folder one will be downloaded for you.
++ If you don't have a b2g_sdk/ folder, one will be downloaded for you.
   This can be problematic if you're offline. You can symlink a
-  b2g-desktop directory to b2g/ in gaia to avoid the download.
+  b2g-desktop directory to b2g_sdk/ in gaia to avoid the download.
```

Comment:
IIRC, user is able to specify various run-time to execute marionette testing. Default is set to mulet which is downloaded in gaia/firefox folder and b2g is set to gaia/b2g (I haven't checked it please correct me if I am wrong).